web/modules/custom/corplite_revision_delete/src/Commands/RevisionDeleteDrushCommands.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\corplite_revision_delete\Commands;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Drush custom commands.
+ */
+class RevisionDeleteDrushCommands extends DrushCommands {
+
+  /**
+   * Core EntityTypeManager instance.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Class constructor.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityManager
+   *   The entity type manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entityManager) {
+    parent::__construct();
+    $this->entityTypeManager = $entityManager;
+  }
+
+  /**
+   * Drush command to delete all revisions before current published revision.
+   *
+   * @command revision-delete
+   */
+  public function revisionDelete(string $option) {
+    $n = 0;
+    if (empty($option)) {
+      $this->io()->write("Please specify a content type to delete revisions on", TRUE);
+      return;
+    }
+    // Get the content type of content revisions that need to be deleted.
+    $nodes = $this->entityTypeManager
+      ->getStorage('node')
+      ->loadByProperties(['type' => $option]);
+
+    // Loop through all the nodes and get their revisions.
+    foreach ($nodes as $nid) {
+      $node_storage = $this->entityTypeManager
+        ->getStorage('node');
```

Comment:
Just for optimisation, I think this 'getStorage' line would be better outside the foreach loop, as we only need to do it once